x.py

```diff
@@ -223,6 +223,60 @@ def clang_tidy(dir: str, jobs: Optional[int], clang_tidy_path: str, run_clang_ti
     run(run_command, *options, *regexes, verbose=True, cwd=basedir)
 
 
+def is_rocky_linux():
```

Comment:
hmmm seems in our CI the rocky linux job works well now.

Reply:
looks like it is needed to do CGO https://github.com/ltagliamonte-dd/kvrocks/actions/runs/14361821853/job/40266547259

Reply:
putting back this change fixes the CI for rocky linux: https://github.com/ltagliamonte-dd/kvrocks/actions/runs/14363316453